recipe/meta.yaml

```
@@ -77,6 +77,24 @@ test:
   requires:
     - pip
 
+outputs:
+  - name: {{ name|lower }}{{ suffix }}
+  - name: {{ name|lower }}{{ suffix }}-cpp
+    script: install-cpp.sh
+    requirements:
+      run:
+        - protobuf
+        - {{ pin_compatible('re2') }}
```

```suggestion
        - {{ pin_compatible('re2') }}
      run_constrained:
        - onnxruntime-cpp <0a0  # [suffix == "-novec"]
```
To avoid installing the vectorized and non-vectorized version at the same time.